atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/TransactionManagerTest.java

```diff
@@ -114,7 +114,7 @@ public void shouldNotMakeRemoteCallsInAReadonlyTransactionIfNoWorkIsDone() {
     }
 
     @Test
-    public void shouldConflictIfImmutableTimestampLockExpiresEvenIfNoWrites() {
+    public void shouldConflictIfImmutableTimestampLockExpiresEvenIfNoWritesOnThoroughSweptTable() {
```

Comment:
Can we add the opposite test - don't conflict if there were reads but not on a thorough sweep table?